src/config.ts

```diff
@@ -25,6 +25,10 @@ export const config = {
         eatRadius: 20,
         maxSpeed: 5,
         minSpeed: 4,
+        upperMaxSpeed: 25,

```

Comment:
I'm wondering if there's a nicer way to do all of this... As it stands there are 6 speed values in config, and effectively another three different values in the HTML (min = 0, max = 100, current = 15). 

As it stands we always set the min speed to 80% of the max speed. Perhaps we can enforce that, and then reduce the number of different variables we need?

Perhaps rename it's speed values to a single `baseSpeed` (set by the slider) and then wherever in code we need a max speed, we use the baseSpeed, and wherever we need a min speed we use 80% of that value?

Reply:
I guess you'd still need a concept of minimumBaseSpeed and maximumBaseSpeed (corresponding to 0 to 100 on the slider), but at least that would get this file down to only 3 speed values rather than 6.